collectors/ebpf.plugin/ebpf.d/filesystem.conf

```diff
@@ -0,0 +1,17 @@
+# The `ebpf load mode` option accepts the following values :
+#  `entry` : The eBPF collector only monitors calls for the functions, and does not show charts related to errors.
+#  `return : In the `return` mode, the eBPF collector monitors the same kernel functions as `entry`, but also creates
+#            new charts for the return of these functions, such as errors.
+#
+# The eBPF collector also creates charts for each running application through an integration with the `apps plugin`.
+# If you want to disable the integration with `apps.plugin` along with the above charts, change the setting `apps` to
+# 'no'.
+#  
+[global]
+    ebpf load mode = entry
+    apps = yes
+    update every = 2
+    pid table size = 32768
+
+[filesystem]
+    ext4dist = yes
```

Comment:
What does mean `dist`?

Reply:
This was a request from product team to simplify comparison with other software.  The `dist` has relationship with `distribution`, because these charts will be histograms in a near future.

Reply:
should we add this as comment? I also didn't understand what it mean until I read this review from @vlvkobal

Reply:
Thank you @underhood, I added a comment.